libraries/rush-lib/src/logic/test/parameterIgnoringRepo/common/config/rush/command-line.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json",
+  "commands": [],
+  "parameters": [
```

Comment:
Include at least one parameter type that takes an argument (string, choice, or integer) and at least one that takes multiple values (string list, or choice list) to have more complete tests